ftlib/commlib/pytorch/impl.py

```diff
@@ -49,10 +50,20 @@ def allreduce(self, data, op="MEAN"):
             size = float(dist.get_world_size())
             data /= size
 
+    @BasicCommLib.register_api
+    def allreduce_np(self, data_np, op="MEAN"):
+        data = torch.from_numpy(data_np)
```

Comment:
Maybe consider merging this with `allreduce` and call `from_numpy` when `data` is numpy data types? This way we can probably merge this into master and it won't affect the existing APIs.

Reply:
Great idea! Please check the new implementation.